From 22f4e0f44afa30c3558dedfd906d5d13e38e435b Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Tue, 6 Mar 2012 16:09:47 +0800
Subject: [PATCH 1137/2508] ENGR00176175 MX6Q: fix typo in cpu op voltage
 setting.

Fix a typo when adding 600M WP, the voltage value is wrong,
it will lead a warnning when change to this WP:

COULD NOT SET GP VOLTAGE!!!!

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index 781e906..f6e5af1 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -38,7 +38,7 @@ static struct cpu_op mx6_cpu_op_1G[] = {
 	{
 	 .pll_rate = 624000000,
 	 .cpu_rate = 624000000,
-	 .cpu_voltage = 11000000,},
+	 .cpu_voltage = 1100000,},
 	 {
 	  .pll_rate = 792000000,
 	  .cpu_rate = 396000000,
-- 
1.7.7.4

